src/Microsoft.OData.Client/EntityDescriptor.cs

```diff
@@ -259,7 +259,7 @@ public ReadOnlyCollection<LinkInfo> LinkInfos
         }
 
         /// <summary>Returns a collection of named binary data streams that belong to the entity.</summary>
-        /// <returns>A <see cref="T:System.Collections.ObjectModel.ReadOnlyCollection`1" /> of <see cref="T:Microsoft.OData.Client.StreamDescriptor" /> objects that are the named binary data streams that belong to the entity.</returns>
+        /// <returns>A <see cref="System.Collections.ObjectModel.ReadOnlyCollection`1" /> of <see cref="Microsoft.OData.Client.StreamDescriptor" /> objects that are the named binary data streams that belong to the entity.</returns>

```

Comment:
I didn't resolve ALL warnings. I will split this across multiple pull requests, as otherwise it will get too big to review. There will still be warnings left after this pull request is completed. Either way, your comment is definitely valid, I added the change.